db/migrate/20210517131953_add_arguments_to_maintenance_tasks_runs.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddArgumentsToMaintenanceTasksRuns < ActiveRecord::Migration[6.0]
+  def change
+    add_column(:maintenance_tasks_runs, :arguments, :text)
```

Comment:
We don't do any check on the size. I guess the good thing about JSON is that if the content is too long, it will not parse and won't risk updating the wrong rows etc. like it could with YAML. I'd like to test this and add a validation if necessary.

Reply:
Good point, we probably want a validation so we can surface in the UI and fail early rather than just not serializing the data. Do we just want to support the maximum size for a `text` column (65,535 bytes => 30000 char length)?

Reply:
We may be able to pull that from the column itself, in case users have tweaked the migration to support their own use case.